### PR TITLE
Added release template.

### PR DESCRIPTION
**Pull request by fredrikbryntesson**

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Is this ready for final review?

**Comment by fredrikbryntesson** [comment]
Yes, please suggest if you want to move the release headline or make it smaller.

**Comment by marcusnaslund** [comment]
Sizes are good. However, you should merge your dependencies part as a subtitle under `How to run`.
Also, we should mention the gcc version we've used, and Windows 7 support as well

File: README.md [hunk 1]
```diff
@@ -10,6 +12,16 @@ To compile, you need the `magic` version of the [Rock compiler](https://github.c
...
 Tests can be run using the `test.sh` script.
 
+### Dependencies
+The following software is necessary to build version X.X.X of ooc-kean:
+* [rock X.X.X](https://github.com/magic-lang/rock/releases/tag/rock_X.X.X)
+* gcc 4.8.4
+
+### Microsoft Windows
+Rock have been built and tested on the following platforms:
+* Built with `MinGW` on Windows 7
```
Comment (marcusnaslund) [inline README.md]: ...using the following instructions: https://github.com/magic-lang/rock/blob/master/Compile-Rock-on-Windows.md

File: README.md [hunk 2]
```diff
@@ -10,6 +12,16 @@ To compile, you need the `magic` version of the [Rock compiler](https://github.c
...
+
+
```
Comment (marcusnaslund) [inline README.md]: Extra blank line

File: README.md (hunk 1 above)
Comment (marcusnaslund) [inline README.md]: Actually, this is all related to rock, not this repo.

Comment (marcusnaslund) [inline README.md]: Maybe write that it's been tested on Ubuntu 14.04 and Windows 7. The rock repo has further details on how to compile on the two platforms.

Comment (marcusnaslund) [inline README.md]: And OS X has some experimental support, it mostly works, according to @thomasfanell

Comment (thomasfanell) [inline README.md]: Indeed, `test.sh` needs some fixing though. OSX bash can't parse it correctly.

**Comment by marcusnaslund** [comment]
Looks like pure perfection now

**Pull request merged**